content/soarian/dstu2/diagnostic/observation.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Observation | Soarian DSTU 2 API 
+---
+ 
+# Observation
+
+* TOC
+{:toc}
+
+## Overview
+
+For the Observation resource query, the API returns observation names and values for elements that the Soarian client has configured as appropriate for the API.  The ONC (Office of the National Coordinator) CCDS (common clinical data set) definition is defined to include at least two types of data relevant to this resource: smoking status and vital signs.  The API will return any data, not marked as erroneous, the client has configured as relevant for the API.  Note that ONC has clarified CCDS sex as the sex recorded at birth and this may be included in the Observation resource if client-configured as part of the social history template.  Data is returned in the sequence defined for these sections.  The configuration of qualifying data (data configured for social history and vital signs sections) is a single configuration that applies to both the API and the Consolidated Clinical Document Architecture (C-CDA) document, social history and vital signs sections, available to the patients. 
```

Comment:
definition is defined to include -> definition includes

Apply above to CarePlan as well and anywhere else